pkg/harvester-manager/machine-config/harvester.vue

```diff
@@ -733,22 +695,35 @@ export default {
       }
     },
 
-    async getEnabledVGpuDevices() {
+    async getAvailableVGpuDevices() {
       const clusterId = get(this.credential, 'decodedData.clusterId');
 
       if (clusterId) {
         const url = `/k8s/clusters/${ clusterId }/v1`;
-        const res = await this.$store.dispatch('cluster/request', { url: `${ url }/${ HCI.VGPU_DEVICE }s` });
-
-        this.vGpuEnabledDevices = (res?.data || [])
-          .filter((v) => v.spec.enabled)
-          .reduce((acc, v) => ({
-            ...acc,
-            [v.id]: {
-              type: VGPU_PREFIX.NVIDIA + v.spec.vGPUTypeName?.replace(' ', '_'),
-              id:   v.id
-            },
-          }), {});
+
+        const vGpus = await this.$store.dispatch('cluster/request', { url: `${ url }/${ HCI.VGPU_DEVICE }s` });
```

Comment:
Could we change `HCI.VGPU_DEVICE` to `HCI.VGPU_DEVICES` in @shell/config/types.js  instead of append 's' here ?

Reply:
I think we can just remove the s . The API supports single/plural entity name:

![image](https://github.com/user-attachments/assets/45dc7d7e-366e-40c3-ae12-a1087bd19c67)